server/internal/server/server.go

```diff
@@ -139,10 +139,6 @@ func fakeAuthInto(ctx context.Context) context.Context {
 
 // organizationRole returns a role that the given user has for the given organization.
 func (s *S) organizationRole(orgID, userID string) v1.OrganizationRole {
-	if !s.enableAuth {
-		return v1.OrganizationRole_ORGANIZATION_ROLE_OWNER
-	}
```

Comment:
I'd like to have this logic placed as outer as possible to keep this kind of primitive functions/methods simple for readability. It might be controversial. Let me know!